.github/workflows/phpunit.yml

```
@@ -0,0 +1,71 @@
+                    MYSQL_ROOT_PASSWORD: wordpress
+                    MYSQL_DATABASE: wordpress
+                    MYSQL_USER: wordpress
+                    MYSQL_PASSWORD: wordpress
+        steps:
+            - uses: actions/checkout@v4
+            - name: Setup Node.js (.node-version)
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20.x
+            - name: Cache multiple paths
+              uses: actions/cache@v3
```

actionlint (1.7.11)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)